data/audio_pad.rodata.s

```diff
@@ -0,0 +1,13 @@
+.include "macro.inc"
+
+# assembler directives
+.set noat      # allow manual use of $at
+.set noreorder # don't insert nops after branches
+.set gp=64     # allow use of 64-bit general purpose registers
+
+.section .rodata
+
+.balign 16
+
+glabel audio_padding
+    .incbin "baserom.z64", 0x44DC0, 0x20
```

Comment:
What is this padding? Is it just 0's? Would it work to define data here of 0's instead of `.incbin`ing? Just an idea

Reply:
It's just padding, yeah.  It's not aligned to a power of two, otherwise I would've used the rom_align directive.  I briefly tried just doing .bytes instead, but this seemed a bit cleaner.

Reply:
Oh, now I can reply here.  Weird.  Anyway, updated this to use fill.

The padding is completely unreferenced by the audio data and tables. It doesn't make the size of the file match a power of two or align to the nearest page table, so there's no obvious reason why these padding bytes exist. It could an unused .rodata section, a side effect of whatever tool Nintendo used to generate these files, or even a separate file that just contains 0s. It's hard to know.